src/app/store.ts

```diff
@@ -1,6 +1,13 @@
 import { combineSlices, configureStore } from '@reduxjs/toolkit';
+import { reducer as currentTodoSlice } from '../features/currentTodo';
+import { reducer as filterSlice } from '../features/filter';
+import { reducer as todosSlice } from '../features/todos';
 
-const rootReducer = combineSlices();
+const rootReducer = combineSlices({
+  currentTodo: currentTodoSlice,
+  filter: filterSlice,
+  todos: todosSlice,
+});
```

Comment:
The function `combineSlices` is incorrect. You should use `combineReducers` from `@reduxjs/toolkit` to combine your reducers. Replace `combineSlices` with `combineReducers`.